Userland/allocate.cpp

```diff
@@ -33,18 +33,19 @@
 
 static void usage(void)
 {
-    printf("usage: allocate [number [unit (B/KB/MB)]]\n");
+    printf("usage: allocate [number [unit (B/kiB/MiB)]]\n");
```

Comment:
Casing (this actually doesn't match with the implementation too ^^')
```suggestion
    printf("usage: allocate [number [unit (B/KiB/MiB)]]\n");
```

Reply:
Looked through all of `git log -SkiB master..HEAD`, hopefully finally got them all.